src/server/server_family.cc

```diff
@@ -895,7 +895,7 @@ error_code ServerFamily::DoSave(bool new_version, Transaction* trans, string* er
     snapshots.resize(1);
 
     ExtendFilenameWithShard(now, -1, &filename);
-    path += filename;
+    path /= filename;
```

Comment:
maybe add a comment saying that `/=` is important because it appends to a directory as opposed to a string.